ushahidi/platform-client
# Refactor maps code

This pull request makes the following changes:
- Rewrite post-view-map
- Rewrite post detail maps
- Vastly simplify Maps helper
- Remove angular-leaflet-directive
- Rewrite location editor
- Rewrite map settings

Test these changes by:
- Load the map view, check it all works fine
- Check map popups work
- Check it works with clustering disabled
- Edit a post, check the location editor all works
- Edit map settings
    - confirm you can move the marker around the map, zoom in/out
    - confirm map updates when changing settings below it

Ping @ushahidi/platform


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/ushahidi/platform-client/416)
